actions/nuxeo-add-to-collection-button.html

```diff
@@ -53,8 +50,11 @@
 
     <dom-if if="[[_isAvailable(document)]]">
       <template>
-        <paper-icon-button icon="[[icon]]" on-click="_toggleDialog" noink></paper-icon-button>
-        <nuxeo-tooltip>[[i18n('addToCollectionButton.tooltip')]]</nuxeo-tooltip>
+        <div class="action" on-click="_toggleDialog">
+          <paper-icon-button icon="[[icon]]" noink></paper-icon-button>
+          <span class="label" hidden$="[[!showLabel]]">[[_label]]</span>
```

Comment:
even if we're saving an `i18n` call I'd rather save the verbosity of having a computed label and go with an expression here (unless we have a dynamic label, ie toggle buttons)

Reply:
But this also means that if the i18n function changes, there'll be way more nodes to update. Also, isn't this something that could be fixed when we refactor actions and perhaps use some sort of [template inheritance](https://www.polymer-project.org/2.0/docs/devguide/dom-template#inherit)?